.github/bin/cleanup-node.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+echo "Space before cleanup"
+df -h
+
+echo "Removing redundant directories"
+sudo rm -Rf /opt/hostedtoolcache/go/
+sudo rm -rf /usr/local/lib/android
```

Comment:
@wendigo 

any reason to have trailing / here and use -R,
and use -r in others ?

Reply:
none